tests/integration/utils/process.py

```diff
@@ -37,8 +37,12 @@ def wait_for_port_subprocess(
 
 def stop_process(proc: Optional[Popen]) -> None:
     if proc:
-        os.kill(proc.pid, signal.SIGKILL)
-        proc.wait(timeout=10.0)
+        try:
+            os.kill(proc.pid, signal.SIGKILL)
+            proc.wait(timeout=10.0)
+        # ProcessLookupError: Raised when the process is already gone
+        except ProcessLookupError:
```

Comment:
I experience this while working on another PR. I'm not 100% sure what was the race, one hypothesis is that the kill was fast enough that wait failed because the process was gone.

Reply:
Why not use `Popen.kill`?

Reply:
No reason, it should be the same :)